jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -186,17 +174,6 @@ def data(self, cls=None):
 
 @viewer_registry("cubeviz-profile-viewer", label="Profile 1D (Cubeviz)")
 class CubevizProfileView(SpecvizProfileView):
-    # Whether to inherit tools from glue-jupyter automatically. Set this to
-    # False to have full control here over which tools are shown in case new
-    # ones are added in glue-jupyter in future that we don't want here.
-    inherit_tools = False
-
-    tools = ['jdaviz:homezoom',
-             'jdaviz:boxzoom', 'jdaviz:xrangezoom',
-             'jdaviz:panzoom', 'jdaviz:panzoom_x',
-             'jdaviz:panzoom_y', 'bqplot:xrange',
-             'jdaviz:selectslice', 'jdaviz:selectline']
-
     # categories: zoom resets, zoom, pan, subset, select tools, shortcuts
     tools_nested = [
```

Comment:
Does `tools_nested` make sense anymore in context of this PR's changes?

Reply:
I _think_ so.  This PR still exposes a property for `viewer.tools` which is a read-only flattened version of this nested list to match the same format that is set as `tools` by the parent classes.  Alternatively we could call the nested list as `tools` but then would probably need to set `inherit_tools=False` to avoid conflicts with upstream logic.